insights/specs/__init__.py

```diff
@@ -22,6 +22,7 @@ class Specs(SpecSet):
     ceph_config_show = RegistryPoint(multi_output=True)
     ceph_df_detail = RegistryPoint()
     ceph_health_detail = RegistryPoint()
+    ceph_log = RegistryPoint(multi_output=True, filterable=True)
```

Comment:
Do we really need `multi_output` here? AFAIK this is used when file path is in multiple places, mostly suitable to conf files but not strictly.

Reply:
@psachin 
Acknowledged. 'multi_output' is not required here. Changes done in latest commit.